contracts/test/ETO/MockETOCommitment.sol

```diff
@@ -43,6 +43,15 @@ contract MockETOCommitment is
         }
     }
 
+    // convenience function for moving all timestampts towards the past
+    // such that the next state transition will occur in delta seconds
+    function _shiftToBeforeNextState(uint8 delta) public {
+        require(delta < 180);
+        uint256 nextTransition = startOfInternal(ETOState(uint(state()) + 1));
+        require(nextTransition != 0 && nextTransition > now + delta);
```

Comment:
please use revert code like `NF_MOCK_INVALID_DELTA` or similar. it will be easier to debug.

Reply:
I thought so too, but I never so any revert-errormessages on any of the rquirements so I didn't. That could be something to do the next we do any changes to the contracts..